src/MusicStore/Components/SystemClock.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace MusicStore.Components
+{
+    /// <summary>
+    /// Provides access to the normal system clock.
+    /// </summary>
+    public class SystemClock : ISystemClock
+    {
+        /// <inheritdoc />
+        public DateTime UtcNow
+        {
+            get
+            {
+                // The clock measures whole seconds only, to have integral expires_in results, and
```

Comment:
Got it.. I'll update the comment as follows:
// The clock measures whole seconds only, and truncates the milliseconds,
// because the millisecond resolutions are inconsistent among various underlying systems.
Since this is only thing I should upate from this PR so far, could you sign it off, if there is no more comments?  I'll update the comment when I check in this PR.